tests/vmi_lifecycle_test.go

```diff
@@ -84,7 +84,8 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
 
-		vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+		vmi = libvmi.NewAlpine()
+		vmi.Namespace = testsuite.GetTestNamespace(vmi)
```

Comment:
This should not be required. We are not suppose to specify the namespace in the spec, but to specify it in the creation command.

I would even consider dropping this BeforeEach completly, it has no value IMO.
With it, if possible, to take out the `vmi` variable that is shared with all tests.

Reply:
I will try to remove the specification of the namespace. 

On dropping it completely, i think there are a number of tests that rely on that alpine VMI being available - so you are suggesting create it in each of these tests separately?